phat/crates/rollup/Cargo.toml

```diff
@@ -25,8 +25,9 @@ ethabi = { version = "18.0.0", default-features = false, features = [
     "rlp",
 ], optional = true }
 
-# for Substrate rollup
-subrpc = { package = "pink-subrpc", version = "0.4.2", default-features = false, optional = true }
+# for Substrate and ink! rollup
+#subrpc = { package = "pink-subrpc", version = "0.4.2", default-features = false, optional = true }
+subrpc = { git = "https://github.com/Phala-Network/phala-blockchain/", package = "pink-subrpc", default-features = false, optional = true }
```

Comment:
Why don't use the published version? Too old?

Reply:
The last version available in the rust package registry doesn't contain my changes done in the milestone 1

Reply:
Got it. Let's make a release once we have finished the project!